release/windows-installer/src/WindowsInstaller.hs

```diff
@@ -13,18 +13,28 @@ import SdkVersion
 
 main :: IO ()
 main = do
-    [installerFile, sdkDir] <- getArgs
-    writeFile installerFile $ nsis $ installer sdkDir
+    [installerFile, sdkDir, logo] <- getArgs
+    writeFile installerFile $ nsis $ installer sdkDir logo
 
-installer :: FilePath -> Action SectionId
-installer sdkDir = do
-    name "DAML SDK installer"
+installer :: FilePath -> FilePath -> Action SectionId
+installer sdkDir logo = do
+    name "DAML SDK"
     outFile "daml-sdk-installer.exe"
+    unsafeInjectGlobal $ "!define MUI_ICON \"" <> logo <> "\""
```

Comment:
Can you use https://hackage.haskell.org/package/nsis-0.3.2/docs/Development-NSIS.html#v:installIcon instead?

Reply:
Ah missed that when browsing the docs, changed.